### PR TITLE
Adding sub, mul, div function for bacardi test.

### PR DESCRIPTION
**Pull request by gogag2**
We added sub, mul, div function to test bacardi project.
Then we will modulrization them with next patch.

ISSUE=#10

### CODE REVIEW

**Review by romandev — CHANGES_REQUESTED** [review]
Patch looks good to me but please update your name and email in AUTHORS file and then include it in this patch.

**Comment by gogag2** [comment]
Could you review it again ? : )

File: AUTHORS [hunk 1]
```diff
@@ -6,5 +6,6 @@
 # For organizations:
 #   Organization <fnmatch pattern>
 
+Byoungkwon Ko <bk_1.ko@samsung.com>
```
Comment (romandev) [inline AUTHORS]: Why don't you use your personal email?

File: examples/calculator.js [hunk 2]
```diff
@@ -22,3 +22,24 @@ assert.equal(calculator.add(3.3, -4.3), -1);
 assert.equal(calculator.add(-3.3, 4.3), 1);
 assert.equal(calculator.add(1.2, 0), 1.2);
 assert.equal(calculator.add(4.5878, 1.1234), 5.7112);
+
+assert.equal(calculator.sub(1, 2), -1);
+assert.equal(calculator.sub(3.3, 4.3), -1);
+assert.equal(calculator.sub(3.3, -4.3), 7.6);
+assert.equal(calculator.sub(-3.3, 4.3), -7.6);
+assert.equal(calculator.sub(1.2, 0), 1.2);
+// <TODO:wrong value> assert.equal(calculator.sub(4.5878, 1.1234), 3.4644);
```
Comment (romandev) [inline examples/calculator.js]: Should be TODO(your id or your name): Blah Blah. Please see: issue link.

**Review by romandev — COMMENTED** [review]
In the commit description, should be ISSUE=#10

**Review by romandev — APPROVED** [review]
LGTM

**Pull request merged**